WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserType.java

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui.media;
+
+public enum MediaBrowserType {
+    BROWSER,                 // browse & manage media
+    EDITOR_PICKER,           // select multiple images or videos to insert into a post
+    FEATURED_IMAGE_PICKER,  // select a single image as a featured image
+    GRAVATAR_IMAGE_PICKER;   // select a single image as a gravatar
```

Comment:
I wonder if we can simplify the types, and just have an enum like this:

```
public enum MediaBrowserType {
    BROWSER,                 // browse & manage media
    MULTIPLE_PICKER,           // select multiple images or videos to insert into a post
    SINGLE_IMAGE_PICKER;        // select a single image as a featured image, a gravatar, etc.

    public boolean isPicker() {
        return this != BROWSER;
    }

    public boolean isBrowser() {
        return this == BROWSER;
    }

    public boolean isSingleImagePicker() {
        return this == SINGLE_IMAGE_PICKER;
    }

    public boolean canMultiselect() {
        return this == MULTIPLE_PICKER;
    }

}

```

(note that the `imagesOnly()` method goes away as it’s the same as `isSingleImagePicker()`)


If we don’t do that above, I think we can still simplify it a bit and at least remove `imagesOnly()` and replace with `isSingleImagePicker()`
(also `imagesOnly` is inconsistent with the form of <verb><state> naming we’re using for the other methods, i.e. is<state> or <can><do_something> like `isPicker` or `canMultiselect`).